.github/workflows/pull_request.yml

```diff
@@ -8,6 +8,25 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
+
+    - uses: actions/checkout@v1
+
+    - name: Setting env vars
+      env:
+        PR: ${{ github.event.number }}
+      run: |
+        UMBREL_OS_VERSION="dev"
+        IMAGE_NAME="umbrel-os-dev"
+        UMBREL_REPO="$(curl -s https://api.github.com/repos/getumbrel/umbrel/pulls/${PR} | jq .head.repo.clone_url | sed 's/"//g')"
+        TMP=$(git show HEAD | head -n 2 | tail -n 1)
```

Comment:
Is this to get the branch? Is that a reliable query?

Running that in a local repo I get:

```
Author: Luke Childs <lukechilds123@gmail.com>
```

If you're getting current branch name here I'd suggest:

```
git branch --show-current
```